### ISSUE TITLE
GLIBCXX_3.4.32' not found for version 1.8.5 - SteamOS 3.5.17

### ISSUE THREAD

**Issue by Hayashi-Tensai** [LogicOverSnacks/glint-release#12]
**Describe the bug**
When launch the application the following error appears

```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: /usr/lib/libstdc++.so.6: version `GLIBCXX_3.4.32' not found (required by /tmp/.mount_Glint-U3qE3O/resources/app/dist/native_modules/build/Release/nodegit.node)
    at process.func [as dlopen] (node:electron/js2c/node_init:2:2214)
    at Module._extensions..node (node:internal/modules/cjs/loader:1343:18)
    at Object.func [as .node] (node:electron/js2c/node_init:2:2214)
    at Module.load (node:internal/modules/cjs/loader:1098:32)
    at Module._load (node:internal/modules/cjs/loader:945:12)
    at c._load (node:electron/js2c/node_init:2:13672)
    at Module.require (node:internal/modules/cjs/loader:1122:19)
    at require (node:internal/modules/helpers:130:18)
    at 27241 (/tmp/.mount_Glint-U3qE3O/resources/app/dist/main.js:2:473199)
    at r (/tmp/.mount_Glint-U3qE3O/resources/app/dist/main.js:14:1621574)
```

**To Reproduce**
1. Launch the app

**Expected behavior**
App opens with no error

**System Information (please complete the following information):**
 - OS: Linux - SteamOS 3.5.17 - AppImage
 - Glint version 1.8.5
 
 
**Additional context**
It was working well for version 1.8.3 and before

**Comment by MikeJerred** [comment]
The problem has been caused be using a newer version of `gcc` to compile dependencies.

For now I have rolled back the release to v1.8.3, pending a fix. You can uninstall Glint, then reinstall [v1.8.3](https://github.com/LogicOverSnacks/glint-release/releases/tag/v1.8.3) to revert.

Alternatively, you can update `libstdc++` on your system e.g. for Ubuntu:
```
sudo add-apt-repository ppa:ubuntu-toolchain-r/test
sudo apt-get update
sudo apt-get install --only-upgrade libstdc++6
```

I will update this issue once fix is in place, and automated tests added for linux packaging!

**Comment by Hayashi-Tensai** [comment]
Ok thanks, I will revert to v1.8.3 for the moment. Also, it would be nice to add a feature to make auto update optional in the case a future update breaks something again, and the old version can be used without being forced to update.

**Comment by MikeJerred** [comment]
Resolved in [v1.8.6](https://github.com/LogicOverSnacks/glint-release/releases/tag/v1.8.6)
